client/src/components/App.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { ajax } from 'jquery';
+import MarketOrder from './MarketOrder.jsx';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      orderType: 'Market Order',
+      stock: {},
+      watch: false
+    }
+    this.getRandomStock = this.getRandomStock.bind(this);
+    this.toggleWatch = this.toggleWatch.bind(this);
+    this.renderView = this.renderView.bind(this);
+  }
+
+  componentDidMount() {
+    this.getRandomStock();
+  }
+
+  toggleWatch(e) {
+    e.preventDefault();
+    this.setState((p) => {
+      return {
+        watch: !p.watch
+      }
+    })
+  }
+
+  renderView() {
+    const { orderType } = this.state;
+
+    if (orderType === 'Market Order') {
+      return (
+        <div className="marketOrder">
+          <MarketOrder stock={this.state.stock}/>
+        </div>
+      );
+    }
+  }
+
+  getRandomStock() {
+    ajax({
+      url: '/stocks',
+      method: "GET",
```

Comment:
May consider switching to single quotes here to appease the linter

Reply:
Thanks, I should probably implement the linter ASAP as well!